api-logs.go

```diff
@@ -42,6 +42,7 @@ type APILogOpts struct {
 	Interval   time.Duration `json:"interval,omitempty"`
 	Origin     log.Origin    `json:"origin,omitempty"`
 	Type       log.APIType   `json:"type,omitempty"`
+	MaxPerNode int           `json:"maxPerNode,omitempty"`
```

Comment:
What is this? I don't see it used anywhere.

Reply:
@Praveenrajmani you mean you have a pending PR that uses this, right ?